### DISCUSSION TITLE
2012 Twitter

### DISCUSSION THREAD

**Discussion by TheToadFan95** [dimdenGD/OldTwitter#694] [Q&A]
Anybody got some personalized CSS to make Twitter look like how it did in 2012? Or maybe, will there be a built-in setting to do it?

**Comment by haydengriz** [comment]
Wrong Department

**Reply by TheToadFan95** [reply]
Where should I say it? New around here.

**Reply by dimdenGD** [reply]
It's a fine place to ask. I highly doubt this will ever be done, at least I'm not planning to.

**Reply by haydengriz** [reply]
Not really but whatever, besides I’m pretty sure this was asked before
